packages/perseus/src/__tests__/renderer-api.test.tsx

```diff
@@ -8,16 +8,19 @@ import TestUtils from "react-dom/test-utils";
 import _ from "underscore";
 import "jest-enzyme";
 
-import {ClassNames, Dependencies, Renderer} from "@khanacademy/perseus";
```

Comment:
This was just wrong. We shouldn't be importing symbols from our own package like this.